tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+envlist = py39-django{22,32,40},py39-django{22,32,40}-swapper
+
+[testenv-django{22,32,40}]
+deps =
+    django22: Django==2.2
+    django32: Django==3.2
+    django40: Django==4.0
+    djangorestframework
+commands = ./runtests.py
+
+[testenv:py39-django{22,32,40}-swapper]
+deps =
+    django22: Django==2.2
+    django32: Django==3.2
+    django40: Django==4.0
+    djangorestframework
+    swapper: swapper==1.3.0
+commands = ./runtests_swapped.py
```

Comment:
It seems there are many duplications with the current Github CI setup, could we use GitHub action to do this?